tox.ini

```diff
@@ -1,20 +1,20 @@
 [tox]
 envlist =
 	py{38,39}-neo4j{4,5}-pandas{1,2}-main-{standard,encrypted},
-	py{38,39}-pandas{1,2}-pyarrow{4,5,6,7,8,9,10,11,12}-main-{standard,encrypted},
-	py310-pandas{1,2}-pyarrow{7,8,9,10,11,12}-main-{standard,encrypted},
-	py311-pandas{1,2}-pyarrow{10,11,12}-main-{standard,encrypted},
+	py{38,39}-pandas{1,2}-pyarrow{4,5,6,7,8,9,10,11,12,13}-main-{standard,encrypted},
+	py310-pandas{1,2}-pyarrow{7,8,9,10,11,12,13}-main-{standard,encrypted},
+	py311-pandas{1,2}-pyarrow{10,11,12,13}-main-{standard,encrypted},
```

Comment:
when can we drop smaller pyarrow versions? 🙈

Reply:
https://pypi.org/project/pyarrow/4.0.0/#history